.pre-commit-config.yaml

```
@@ -0,0 +1,27 @@
+repos:
```

```suggestion
exclude: ".*\\.asdf$"

repos:
```
This was added in romancal in https://github.com/spacetelescope/romancal/pull/1161 when it was noticed that pre-commit treats ASDF files as text and will strip whitespace from binary portions of the file.